mirrord/cli/src/main.rs

```diff
@@ -301,6 +309,60 @@ fn exec(args: &ExecArgs, progress: &TaskProgress) -> Result<()> {
     Err(anyhow!("Failed to execute binary"))
 }
 
+/// Returns a list of (pod name, [container names]) pairs.
+async fn get_kube_pods(namespace: Option<&str>) -> Result<HashMap<String, Vec<String>>> {
```

Comment:
Is `kube` necessary here? Are there any other kinds of pods?

Reply:
kube just qualifies that this logic will deal with k8s, there are not other kind of pods. we can remove this, lmk